static/app/components/stream/group.tsx

```diff
@@ -310,6 +310,11 @@ class StreamGroup extends React.Component<Props, State> {
     const {data} = this.state;
     const {statusDetails, count} = data as GroupReprocessing;
     const {info, pendingEvents} = statusDetails;
+
+    if (!info) {
```

Comment:
this is not really part of the scope of this PR, but since I was having an error locally and this is an easy fix decided to push with the rest of the code

Reply:
Are the types for `GroupReprocessing` incorrect? I would have expected typescript to complain about the unhandled null/undefined on line 318.

Reply:
sure. added type 😉